### ISSUE TITLE
Good idea, very poor execution

### ISSUE THREAD

**Issue by master0v** [kepon85/PvMonit#7]
what kind of error messaging is this?

<img width="343" alt="Screen Shot 2020-04-01 at 12 09 31 PM" src="https://user-images.githubusercontent.com/18746987/78160324-c49b4e80-7411-11ea-86d5-ef79d17f2421.png">

**Comment by kepon85** [comment]
Yes, moreover, PvMonit is only available in French at the moment.

Do you have a valid config.yaml file for your error? : https://github.com/kepon85/PvMonit/blob/master/INSTALL.md#la-base--le-socle

`cp config-default.yaml config.yaml`

And edit for you...
